src/components/Modal/modal.module.scss

```
@@ -0,0 +1,27 @@
+.modalContainer {
+  width: 60vw;
+  height: 60vh;
+  padding-left: 30px;
+  background-color: rgb(255, 255, 255);
+}
+.background {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin: 0;
+  padding: 0;
+  position: absolute;
+  top: -50px;
+  left: -50px;
+  width: 100vw;
+  height: 100vh;
+  background-color: rgba(255, 255, 255, 0.3);
+  backdrop-filter: blur(0.5rem);
+}
+
+.closeIcon {
+  position: absolute;
+  top: 6%;
+  right: 6%;
```

A la hora de posicionar un elemento, si es una medida fija, es mejor usar pixeles que porcentaje, a menos que específicamente se necesite para algo en particular (que esta a la mitad, a un cuarto, etc)
```suggestion
  top: 15px; /** O Cualquier otro valor */
  right: 15px;
```